lib/indico.rb

```diff
@@ -85,10 +85,15 @@ def self.relevance(text, queries, config = nil)
       config = Hash.new()
     end
     config[:queries] = queries
+    config[:synonyms] = false
     return api_handler(text, 'relevance', config)
   end
 
   def self.text_features(text, config = nil)
+    if config.nil?
+      config = Hash.new()
+    end
+    config[:synonyms] = false
```

Comment:
I think we want `config[:synonyms] = false` in the if statement for both of these, otherwise it will overwrite the param if a user set it.